README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
Confusion matrix:
A matrix Ci,j where i is the actual classification group and j is the predicted classification group:
For example

          | Cat | Dog | Bird
Cat    | 3     | 2      | 0
Dog   | 1     | 4      | 0
Bird    | 0    | 0      | 11

Anything outside the diagonal is an 'error'
Then there's the table of confusion, which is a binary analysis of a single class and describes accuracy in terms of true positives, false positives, true negatives, false negatives
True is 'good' false is 'bad'
For the dog above:

        T   |   F
P   |  4   | 2
      ---------
N   |  14  |  1